bsip-0062.md

```diff
@@ -0,0 +1,43 @@
+ ```
+  BSIP: 0062
+  Title: Close short position 
+  Authors:
+    Stefan Schießl stefan.schiessl@blockchainprojectsbv.com
+    Jerry Liu bitcrab@qq.com
+  Status: Draft
+  Type: Protocol
+  Created: 2019-09-14
+  Worker: TBD
+  ```
+
+# Abstract
+Closing a short position comes with a risk of getting margin called if no spare funds are available to buy the long and reduce the debt to zero. This BSIP introduces a way to put the short position directly on the market.
+
+# Rationale
+Closing a short position with a healthy CR should come with no risk of getting margin called.
+
+# Specification
+Introduce a way that user can directly place orders to sell collateral in a margin position, and the received SmartCoins will be used to reduce the debt automatically.
+
+User needs to define the amount to sell and amount to receive while placing the sell orders, and the 
+price should be greater or equal to margin call price/MCR, to ensure the deposit position CR will not be reduced after the filling of the sell orders.
+
+When a debt position is margin called with close short orders, 2 ways are possible to handle this:
```

Comment:
Please specify only one way. The other can be part of a "Discussion" section, together with a reason why it wasn't chosen.